apps/cowswap-frontend/src/modules/trade/types/TradeType.ts

```diff
@@ -0,0 +1,12 @@
+import { RoutesValues } from 'common/constants/routes'
+
+export enum TradeType {
+  SWAP = 'SWAP',
+  LIMIT_ORDER = 'LIMIT_ORDER',
+  ADVANCED_ORDERS = 'ADVANCED_ORDERS',
+}
```

Comment:
Isn't this defined already somewhere else?

Reply:
Unfortunately yes :( 
We currently have `UiOrderType` and `TradeType` in widget lib.
I hope one day we will consolidate them.
P.s. the type in `modules/trade` wasn't added in this PR, it just was relocated from another file.